packages/state-utils/package.json

```diff
@@ -8,7 +8,14 @@
 	"sideEffects": true,
 	"main": "dist/cjs/index.js",
 	"module": "dist/esm/index.js",
-	"calypso:src": "src/index.ts",
```

Comment:
Did you intend to remove this line? I see we keep it in other files.

Reply:
In the very first version I was _replacing_ `calypso:src` with `exports` but then later decided to keep all the existing fields and just _add_ a new one. And I forgot to fix a few packages: `design-picker`, `popup-monitor` and `state-utils`. I pushed a fix now. Thanks for spotting this!